.gitignore

```diff
@@ -1,2 +1,4 @@
-
+build/**
 *.user
+**/*.swp
+output/**
```

Comment:
I would prefer not to ignore this folder completely because it contains the .pro file for configuring with Qt as well as a Matlab/Octave script for plotting the data in the output csv files. To target only the data files, this line could be changed to
```
output/*.dat
output/*.csv
```